web-client/src/pages/PhoneNumberPage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import CenteredCard from '../components/CenteredCard/CenteredCard';
+import GradientBackground from '../components/GradientBackground/GradientBackground';
+import PhoneNumber from '../components/PhoneNumber/PhoneNumber';
+
+const PhoneNumberPage: React.FC = () => (
+  <GradientBackground>
+    <CenteredCard>
+      <PhoneNumber />
+    </CenteredCard>
+  </GradientBackground>
+);
+
+export default PhoneNumberPage;
```

Comment:
This should be inside of /pages/routes since it is indeed a route. You can go ahead and add this as a route to ContentPage.tsx